### PR TITLE
Add tests for actions on linodes

### PR DESCRIPTION
**Pull request by SirCmpwn**
Will finish this up after the weekend. All that's left is to flesh out the tests for creating a Linode.

### CODE REVIEW

**Comment by SirCmpwn** [comment]
Ready for review/merge.

File: test/linodes/actions/create.spec.js [hunk 1]
```diff
@@ -0,0 +1,198 @@
...
+import { mock_context } from '~/../test/mocks';
+import { pushPath } from 'redux-simple-router'
+
+describe("linodes/actions/create", () => {
+  describe("changeSourceTab", () => {
+    it("should return a CHANGE_SOURCE_TAB action", () => {
+      expect(actions.changeSourceTab(1))
+        .to.deep.equal({
+          type: actions.CHANGE_SOURCE_TAB,
+          tab: 1
```
Comment (eatonphil) [inline test/linodes/actions/create.spec.js]: Styleguide says always use trailling commas

**Comment by SirCmpwn** [comment]
Pushed minor fix.

File: test/linodes/actions/create.spec.js (hunk 1 above)
Comment (SirCmpwn) [inline test/linodes/actions/create.spec.js]: Really? Fuck trailing commas.

**Comment by eatonphil** [comment]
This stuff is changing so rapidly. I really don't think it makes sense to use this or continue writing tests. I guess the worst it can get as that you just have to delete some existing tests and write some more new ones. idk

**Comment by SirCmpwn** [comment]
I don't think it'll be a big deal since the actual reducers/actions behind this flow will probably remain the same, even if the UI changes dramatically.

File: test/linodes/actions/create.spec.js (hunk 1 above)
Comment (eatonphil) [inline test/linodes/actions/create.spec.js]: Their reasoning was that it stops some stupid diffs from showing up when you reorder or add new lines.

Comment (SirCmpwn) [inline test/linodes/actions/create.spec.js]: I guess. I'll change it.

Comment (SirCmpwn) [inline test/linodes/actions/create.spec.js]: On second thought, I'll just introduce a linter in another pull request and do style enforcement in one fell swoop.

**Comment by eatonphil** [comment]
Cool, well looks good to me. And a linter test PR will be awesome to see as well. Ready to merge?

**Comment by SirCmpwn** [comment]
Ready to merge.

**Pull request merged**